pkg/urbit/jets/tree.c

```diff
@@ -2114,8 +2114,75 @@ static c3_c* _k140_ha[] = {
   0
 };
 
+
+//  TODO: probably need different ha hashes
+
+static u3j_core _a50_two__by_d[] =
+  { { "apt", 7, _140_two__by_apt_a, 0, _140_two__by_apt_ha },
+    { "del", 7, _140_two__by_del_a, 0, _140_two__by_del_ha },
+    { "get", 7, _140_two__by_get_a, 0, _140_two__by_get_ha },
+    { "has", 7, _140_two__by_has_a, 0, _140_two__by_has_ha },
+    { "put", 7, _140_two__by_put_a, 0, _140_two__by_put_ha },
+    {}
+  };
+
+static u3j_core _a50_two__in_d[] =
+  { { "apt", 7, _140_two__in_apt_a, 0, _140_two__in_apt_ha },
+    { "del", 7, _140_two__in_del_a, 0, _140_two__in_del_ha },
+    { "put", 7, _140_two__in_put_a, 0, _140_two__in_put_ha },
+    {}
+  };
+
+u3j_core _a50_d[] =
+{ { "add", 7, _140_one_add_a, 0, _140_one_add_ha },
+  { "dec", 7, _140_one_dec_a, 0, _140_one_dec_ha },
+  { "div", 7, _140_one_div_a, 0, _140_one_div_ha },
+  { "dvr", 7, _140_one_dvr_a, 0, _140_one_dvr_ha },
+  { "gte", 7, _140_one_gte_a, 0, _140_one_gte_ha },
+  { "gth", 7, _140_one_gth_a, 0, _140_one_gth_ha },
+  { "lte", 7, _140_one_lte_a, 0, _140_one_lte_ha },
+  { "lth", 7, _140_one_lth_a, 0, _140_one_lth_ha },
+  { "mod", 7, _140_one_mod_a, 0, _140_one_mod_ha },
+  { "mul", 7, _140_one_mul_a, 0, _140_one_mul_ha },
+  { "sub", 7, _140_one_sub_a, 0, _140_one_sub_ha },
+
+  { "bex", 7, _140_two_bex_a, 0, _140_two_bex_ha },
+  { "cat", 7, _140_two_cat_a, 0, _140_two_cat_ha },
+  { "can", 7, _140_two_can_a, 0, _140_two_can_ha },
+  { "con", 7, _140_two_con_a, 0, _140_two_con_ha },
+  { "cut", 7, _140_two_cut_a, 0, _140_two_cut_ha },
+  { "dis", 7, _140_two_dis_a, 0, _140_two_dis_ha },
+  { "dor", 7, _140_two_dor_a, 0, _140_two_dor_ha },
+  { "end", 7, _140_two_end_a, 0, _140_two_end_ha },
+  { "gor", 7, _140_two_gor_a, 0, _140_two_gor_ha },
+  { "lsh", 7, _140_two_lsh_a, 0, _140_two_lsh_ha },
+  { "met", 7, _140_two_met_a, 0, _140_two_met_ha },
+  { "mix", 7, _140_two_mix_a, 0, _140_two_mix_ha },
+  { "mor", 7, _140_two_mor_a, 0, _140_two_mor_ha },
+  { "mug", 7, _140_two_mug_a, 0, _140_two_mug_ha },
+  { "muk", 59, _140_two_muk_a, 0, _140_two_muk_ha },  //  TODO: why 59?
```

Comment:
The short answer is that the `+muk` pins the `|fe` core *outside* of itself. The long answer is that but a lot funnier ...